app/src/ui/blank-slate/blank-slate.tsx

```diff
@@ -103,7 +103,7 @@ interface IBlankSlateState {
 export class BlankSlateView extends React.Component<
   IBlankSlateProps,
   IBlankSlateState
-> {
+  > {
```

Comment:
@sr98bj thanks for starting this off! It looks like there's some formatting changes that have been introduced by the editor. To tidy these up, you can run `yarn lint:fix` locally and commit the changes.

Reply:
Okay, Ill do that now and resubmit

Reply:
@sr98bj you can just make new commits to the branch and push those to your fork - that will refresh this pull request without needing to submit a new one...